### ISSUE TITLE
Solid work, but how do I trigger a custom python script (w/ imports?)

### ISSUE THREAD

**Issue by OluAgunloye** [danielgerlag/conductor#21]
Thanks for this Daniel, will build some really cool stuff on it!
In order to do that though, see subject: can we call our own custom scripts that may need their own Python package imports?

**Comment by OluAgunloye** [comment]
@danielgerlag 
Thought I'd bump this, for a future reply. If this is not on the roadmap, what obstacles stop this from being a reality? Would like work on a branch that does this as a PR.

**Comment by danielgerlag** [comment]
You can currently import any system package.  Are you looking to import 3rd party modules?

**Comment by OluAgunloye** [comment]
> You can currently import any system package. Are you looking to import 3rd party modules?
Yes. Also, where can I find some documentation on how to do this, or how to import the system packages?

**Comment by danielgerlag** [comment]
I tried it like this
```
import datetime
result = str(datetime.datetime.now())
```

**Comment by OluAgunloye** [comment]
> I tried it like this
> 
> ```
> import datetime
> result = str(datetime.datetime.now())
> ```
@danielgerlag 
You did this as a custom script then? You created your own step? Or how else would this be done?

**Comment by OluAgunloye** [comment]
@danielgerlag 
Hey Daniel, 

Still stuck on this part actually. The dateTime/Timespan still giving lots of issues.

**Comment by OluAgunloye** [comment]
> I tried it like this
> 
> ```
> import datetime
> result = str(datetime.datetime.now())
> ```

Hey Daniel, when you have time could use some more elaboration on this. Tried just using the built in `System.Timespan()` method to set an interval in any of your workflow steps that accepted an interval. The  IronPython interpretation got in the way. I genuinely wish to know how you suggest we set the interval fields in these workflows, as accessing a datetime object through JSON is not flying!
